src/app/features/issue/providers/gitlab/gitlab-common-interfaces.service.ts

```diff
@@ -66,10 +64,12 @@ export class GitlabCommonInterfacesService implements IssueServiceInterface {
 
     const cfg = await this._getCfgOnce$(task.projectId).toPromise();
     const issue = await this._gitlabApiService.getById$(+task.issueId, cfg).toPromise();
-
+    const user = await this._issueCacheService.projectCache<GitlabUser>(task.projectId, 'GITLAB_USER', duration({days: 1}), () => {
```

Comment:
Can you maybe explain in a sentence or two why you switched from the current method of providing a custom string, to making an additional request? Is this just for convenience or is there another reason?

Reply:
It will not make another request if the user is in the cache. If it's expired or not in the cache it will fetch the user. The static "login" field is a bad concept:
1. if the user deleted and another user created with the same login name it will make the wrong result (example comment filtering) that's why I changed username/login name to id.
2. minimalize the user input is the right way I think.